ledger-tool/src/main.rs

```diff
@@ -2928,7 +2934,7 @@ fn main() {
                     on_halt_store_hash_raw_data_for_debug: arg_matches
                         .is_present("halt_at_slot_store_hash_raw_data"),
                     // ledger tool verify always runs the accounts hash calc at the end of processing the blockstore
-                    run_final_accounts_hash_calc: true,
```

Comment:
Can you please rip out the comment on the line above as well that would now be incorrect